uiowa/uiowa
# i3559 - User Role Permissions

Closes #3559 


# How to test


1. Run ‘blt ds —site=sandbox.uiowa.edu’
2. Run ‘drush @sandbox.local uli’
3. Go to https://sandbox.local.drupal.uiowa.edu/admin/people/create and create a ‘test’ user with the webmaster role selected
    1. Change the author of a piece of content and make sure it saves without errors and updates accordingly
4. Go to https://sandbox.local.drupal.uiowa.edu/admin/people/create 
    1. Create another user or change this users perms to ‘Publisher’ 
        1. Go to https://sandbox.local.drupal.uiowa.edu/admin/content and click on the operations dropdown arrow for the content -> shouldn’t see the ‘delete’ option 
        2. Edit a piece of content -> same situation
        3. Repeat with the ‘Editor’ role and make sure you get the same results

I also had to update the role perms for custom sites - I believe I got all that have any split user perms - make sure:
1.  I removed right to delete any custom content for editors/publishers 
2. I granted webmasters the ability to change the ‘authored by’ field for all custom content types as well please
